benchmark/acsylla_benchmark.py

```diff
@@ -167,7 +167,7 @@ async def main():
         "--concurrency", help="Number of concurrency clients, by default 32", type=int, default=32,
     )
     parser.add_argument(
-        "--duration", help="Test duration in seconds, by default 60", type=int, default=60,
+        "--duration", help="Test duration in seconds, by default 60", type=int, default=10,
```

Comment:
Should we keep the 60 as it is described by the help statment?

Reply:
of course I made this commit by mistake